exchange/exchange_test.go

```diff
@@ -1325,6 +1325,267 @@ func TestUpdateRejections(t *testing.T) {
 	assert.Containsf(t, rejections, "bid rejected [bid ID: bid_id2] reason: some reason 2", "Rejection message did not match expected")
 }
 
+func TestApplyDealSupport(t *testing.T) {
+	bidderName1 := openrtb_ext.BidderName("validbase")
+	bidderName2 := openrtb_ext.BidderName("validmultiple2")
+	bidderName3 := openrtb_ext.BidderName("minnotmet")
+	bidderName4 := openrtb_ext.BidderName("invalidconfig")
+	bidderName5 := openrtb_ext.BidderName("nodeal")
+	bidderName6 := openrtb_ext.BidderName("invalidconfig2")
+
+	bidRequest := &openrtb.BidRequest{
+		ID: "some-request-id",
+		Imp: []openrtb.Imp{
+			{
```

Comment:
Why does this test need 6 imps? If each one is intended to be a unique test case, consider breaking it down using table driven tests. Same feedback for the other tests in this file.

Reply:
Refactored all added unit tests to use the table driven test approach. Also removed a couple test cases that were redundant.